pkg/operator/controller/controller_dns_configmap.go

```diff
@@ -347,6 +365,28 @@ func coreDNSLogLevel(dns *operatorv1.DNS) string {
 	return "class error"
 }
 
+// coreDNSCache reads the TTL values set in spec.cache and returns integer representations of those values in seconds.
+// This returns uint32 values because CoreDNS won't allow setting the maxTTLs to 0 and there's no use case for setting a
+// negative maxTTL value.
+func coreDNSCache(dns *operatorv1.DNS) (positiveTTL, negativeTTL uint32) {
+	configuredPosTTL := uint32(dns.Spec.Cache.PositiveTTL.Seconds())
```

Comment:
uint32(math.Round(dns.Spec.Caceh.PositiveTTL.Seconds())) might give a better result.

Reply:
Fixed in the latest push